src/Illuminate/Mail/Transport/SparkPostTransport.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Illuminate\Mail\Transport;
+
+use Swift_Mime_Message;
+use GuzzleHttp\ClientInterface;
+
+class SparkPostTransport extends Transport
+{
+    /**
+     * Guzzle client instance.
+     *
+     * @var \GuzzleHttp\ClientInterface
+     */
+    protected $client;
+
+    /**
+     * The SparkPost API key.
+     *
+     * @var string
+     */
+    protected $key;
+
+    /**
+     * Create a new SparkPost transport instance.
+     *
+     * @param  \GuzzleHttp\ClientInterface  $client
+     * @param  string  $key
+     * @param  string  $domain
+     * @return void
+     */
+    public function __construct(ClientInterface $client, $key)
+    {
+        $this->client = $client;
+        $this->key = $key;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function send(Swift_Mime_Message $message, &$failedRecipients = null)
+    {
+        $this->beforeSendPerformed($message);
+
+        $recipients = $this->getRecipients($message);
+
+        $message->setBcc([]);
+
+        $options = [
+            'headers' => [
+                'Authorization' => $this->key,
+            ],
+            'json' => [
+                'recipients' => $recipients,
+                'content' => [
+                    'email_rfc822' => $message->toString(),
+                ],
+            ],
+        ];
+
+        return $this->client->post('https://api.sparkpost.com/api/v1/transmissions', $options);
+    }
+
+    /**
+     * Get all the addresses this message should be sent to.
+     *
+     * Note that SparkPost still respects CC, BCC headers in raw message itself.
+     *
+     * @param  \Swift_Mime_Message $message
+     * @return array
+     */
+    protected function getRecipients(Swift_Mime_Message $message)
```

Comment:
Similar but not the same and yes, is copied from Mandrill and changed here: https://github.com/laravel/framework/pull/12579/files#diff-a92e9ec1ecc36811d8edb01394beb2f6R88